druntime/src/core/internal/vararg/aarch64.d

```diff
@@ -26,7 +26,7 @@ import core.stdc.stdarg : alignUp;
 //@nogc:    // Not yet, need to make TypeInfo's member functions @nogc first
 nothrow:
 
-extern (C++, std) struct __va_list
+extern (C++, std) struct __va_list_tag
```

Comment:
Wrong (mangling matters): https://github.com/ARM-software/abi-aa/blob/main/aapcs64/aapcs64.rst#1015definition-of-va_list

Reply:
huh, I made it match sysv_x64.d